packages/repository-tests/src/crud/relations/acceptance/has-many-through.relation.acceptance.ts

```diff
@@ -205,7 +205,37 @@ export function hasManyThroughRelationAcceptance(
       expect(cartItems).have.length(0);
     });
 
-    //FIXME(Agnes): should be able to do deletion based on filters
+    it('can delete instances based on the filter', async () => {
```

Comment:
`can delete instances based on the filter` -> `deletes instances based on the filter`

Please use this format on all test descriptions.